src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
+
+import Html exposing (..)
+
+
+{-
+
+   This module handles the view rendering for the table of players
+   and everything relating to that.
+
+-}
+
+
+type Model
+    = Model
```

Comment:
Yeah, it's just empty for this spike - just enough to get the code in place with the main module.